cypress/integration/brewery/Router - wrong url.spec.js

```diff
@@ -1,14 +1,27 @@
 import { ErrorBanner, Login } from '../../commons/actions';
 import { apiUtils as api } from '../../commons/utils';
+import { setup } from '../../commons/utils/setup';
+import { stub } from '../../commons/services/stubbing';
 
 describe('Sharing with wrong URL', () => {
+  before(() => {
+    setup.initCypressAndStubbing();
+    stub.start({
+      GET_DATASETS: true,
+      GET_SCENARIOS: true,
+    });
+  });
+  after(() => {
+    stub.stop();
+  });
   it("can display error banner when scenario doesn't exist", () => {
     cy.clearLocalStorageSnapshot();
     cy.visit('scenario/invalidurl');
```

Comment:
When the webapp finds out that the scenario does not exist, does it trigger a findScenarioById request on the first scenario of the scenarios list ? If it is the case maybe we should stub this request too to prevent a call to the back-end (especially if GET_SCENARIOS is enabled in the stubbing, we well query the back-end for a non-existing scenario)

Reply:
No, the webapp doen't trigger a findScenarioById, I think it's because we fired this API call and currentScenario is chosen from getAllScenarios list (and this list is stubbed). I tried to stubb the findScenarioById for invalidurl (to search the scenario in the fake workspace) but with no success so this is a real API call. I'll search better today :slightly_smiling_face: